core/deployment/src/main/java/io/quarkus/deployment/builditem/LaunchModeBuildItem.java

```diff
@@ -57,4 +60,12 @@ public boolean isAuxiliaryApplication() {
     public Optional<DevModeType> getAuxiliaryDevModeType() {
         return auxiliaryDevModeType;
     }
+
+    /**
+     * If this is a test. Dev mode tests don't launch with a launch mode TEST, so this
+     * can be used to determine if we are in a dev mode test.
+     */
+    public boolean isTest() {
```

Comment:
This is kinda confusing to be honest. Maybe we can rename this to something more descriptive?